src/core/p5.Renderer.js

```diff
@@ -216,10 +216,10 @@ p5.Renderer.prototype.text = function(str, x, y, maxWidth, maxHeight) {
   let jj;
   let line;
   let testLine;
+  let currentLineLength;
   let testWidth;
   let words;
   let totalHeight;
-  let shiftedY;
```

Comment:
thanks for your review @outofambit, but I think I made a mistake here... adding this shiftedY variable was part of the original PR that I'm reverting part of, but doesn't have to do with my fix, so I'm just going to push a small change to not touch this before this gets merged :)

Reply:
tbh i just did a line by line comparison of this against the original PR :)

thanks for catching that!

Reply:
I did the same and didn't see it until after your approval either 😅 just pushed the fix, should be good 2 go now